Sources/PublicInterface/Session.swift

```diff
@@ -40,12 +40,14 @@ public struct Session: Codable {
         public let description: String?
         public let icons: [URL]
         public let url: URL
+        public let scheme: String?
```

Comment:
@pedrouid should `scheme` be part of ClientMeta? In specs I don't see it.
Imo if some client needs to store scheme for session it can be done out of the protocol.